HACKING.md

```diff
@@ -106,6 +106,47 @@ otherwise an official image can be run with
 
 FAF should be available at ```http://localhost:8080/faf/```
 
+### Reporting into FAF
+1. Set up your server URL for libreport
+
+    Insert `URL = http://localhost:8080/faf` into `/etc/libreport/plugins/ureport.conf`
+2. Create crash
+
+    Make sure you have ABRT running and that you have autoreporting enabled (`abrt-auto-reporting enabled`)
+    Install package `will-crash` and then execute `will_abort`
+
+This should be enough, so in a few minutes (reports are processed every few minutes) you should see
+spike in your FAF. However there are a few places where it can go wrong, therefore following
+troubleshooting:
+
+
+
+1. Was the problem caught by ABRT on your local machine?
+
+    Check output of `abrt-cli ls`. If you dont see any problems there, probably ABRT is not running on your machine. Try reading https://abrt.readthedocs.io/en/latest/ .
```

Comment:
- s/probably ABRT is not running/ABRT is probably not running/
- s/Try reading /Check https://abrt.readthedocs.io/en/latest/ for more information/